client/me/security-2fa-sms-settings/index.jsx

```diff
@@ -135,28 +124,24 @@ const Security2faSMSSettings = createReactClass( {
 				validation: phoneNumber.validation,
 			},
 		} );
-	},
-
-	onSubmitResponse: function ( error ) {
-		this.setState( { submittingForm: false } );
+	};
 
+	onSubmitResponse( error ) {
```

Comment:
Side note: the `on*` can confuse someone that it's an event handler when it's actually not. Should we rename it now that we're touching here?

Reply:
Agreed. Changed here https://github.com/Automattic/wp-calypso/pull/49000/commits/e9e4c328add740e00c2bbbe9aca25aa0becf9149